src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs

```diff
@@ -389,18 +392,17 @@ internal IEnumerable<object> GetCustomAttributesRecursively(MemberInfo attribute
         }
 
         /// <summary>
-        /// Gets the custom attributes on the assembly of a member info
+        /// Gets the custom attributes on an assembly
         /// NOTE: having it as separate virtual method, so that we can extend it for testing.
         /// </summary>
-        /// <param name="memberInfo">The member to inspect.</param>
+        /// <param name="assembly">The assembly to inspect.</param>
         /// <param name="type">The attribute type to find.</param>
         /// <returns>Custom attributes defined.</returns>
-        internal virtual Attribute[] GetCustomAttributeForAssembly(MemberInfo memberInfo, Type type)
+        internal virtual Attribute[] GetCustomAttributeForAssembly(Assembly assembly, Type type)
```

Comment:
GetCustomerAttribute*() methods usulally takes MemberInfo() as argument. Please consider keeping it consistent.